src/main/java/org/kuali/kfs/pdp/businessobject/PaymentGroup.java

```diff
@@ -1107,21 +1107,22 @@ public String getStreet() {
     }
     
     public void validateVendorIdAndCustomerInstitutionIdentifier() {
```

Comment:
I was able to eliminate this code here by updating the calling code. However the setPayeeId function is called by XML configuration.